internal/cmd/argus/scrape-config/describe/describe.go

```diff
@@ -0,0 +1,181 @@
+	"github.com/stackitcloud/stackit-sdk-go/services/argus"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	jobNameArg = "JOB_NAME"
+
+	instanceIdFlag = "instance-id"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	JobName    string
+	InstanceId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("describe %s", jobNameArg),
+		Short: "Shows details of a scrape configuration from an Argus instance",
+		Long:  "Shows details of a scrape configuration from an Argus instance.",
+		Args:  args.SingleArg(jobNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of a scrape configuration with name "my-config" from Argus instance "xxx"`,
+				"$ stackit argus scrape-config describe my-config --instance-id xxx"),
+			examples.NewExample(
+				`Get details of a scrape configuration with name "my-config" from Argus instance "xxx" in a table format`,
+				"$ stackit argus scrape-config describe my-config --output-format pretty"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd, args)
+			if err != nil {
+				return err
+			}
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("read scrape configuration: %w", err)
+			}
+
+			return outputResult(p, model.OutputFormat, resp.Data)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), instanceIdFlag, "Instance ID")
+
+	err := flags.MarkFlagsRequired(cmd, instanceIdFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
+	jobName := inputArgs[0]
+
+	globalFlags := globalflags.Parse(cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	return &inputModel{
+		GlobalFlagModel: globalFlags,
+		JobName:         jobName,
+		InstanceId:      flags.FlagToStringValue(cmd, instanceIdFlag),
+	}, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *argus.APIClient) argus.ApiGetScrapeConfigRequest {
+	req := apiClient.GetScrapeConfig(ctx, model.InstanceId, model.JobName, model.ProjectId)
+	return req
+}
+
+func outputResult(p *print.Printer, outputFormat string, config *argus.Job) error {
+	switch outputFormat {
+	case globalflags.PrettyOutputFormat:
+
+		saml2Enabled := "Enabled"
+		if config.Params != nil {
+			saml2 := (*config.Params)["saml2"]
+			if len(saml2) > 0 && saml2[0] == "disabled" {
+				saml2Enabled = "Disabled"
+			}
+		}
+
+		targets := []string{}
+		for _, target := range *config.StaticConfigs {
+			targetFmt := ""
+			if target.Labels != nil {
+				// make map prettier
+				for k, v := range *target.Labels {
```

Comment:
This seems too complex. I would make it simpler, something like this:
```
var targetLabels []string
targetLabelsString = "N/A"
for k, v := range *target.Labels {
    targetLabels = append(targetLabels, fmt.Sprintf("%s:%s", k, v))
}
if targetLabels != nil {
   targetLabels = strings.Join(targetLabels, ",")
}
targetUrls := "N/A"
if target.Targets != nil {
  targetUrls = *target.Targets
}

(...)

targets := fmt.Sprintf("labels: %s\nurls:%s", targetLabelsString, targetUrls)

```

Check what the Portal shows when labels or urls are empty (I only checked for labels)

Reply:
For labels it shows the "N/A" you gave, for the urls, at least one is required so it is never shown. (I added an initial value for the string of "N/A" instead of "" for uniformity.
Looks like this now:
![image](https://github.com/stackitcloud/stackit-cli/assets/33052643/cbae13ca-853e-479e-b77e-1fdb532be6e8)

Reply:
I am also unsure about the TARGET 1, TARGET 2, etc. since targets are not numbered or named. 
Alternative would be to just print TARGET, TARGET, etc.
Also just print TARGETS in the first one and then have the first column be empty.
Any suggestions?

Reply:
For now we could make it TARGET #1, TARGET #2

Then we can implement a way to merge rows only on the first column, keeping the separator on the others